src/datreant/core/collections.py

```diff
@@ -439,6 +439,17 @@ def relpaths(self):
         return [member.relpath for member in self]
 
     @property
+    def parents(self):
```

Comment:
So a View and a Bundle are the same thing (just the content they have are different)?